addons/web_editor/static/src/js/wysiwyg/wysiwyg_snippets.js

```diff
@@ -17,7 +17,8 @@ Wysiwyg.include({
             var self = this;
             this.editor = new (this.Editor)(this, this.options);
             this.$editor = this.editor.rte.editable();
-            await this.editor.prependTo(this.$editor[0].ownerDocument.body);
+            this.$body = (this.$editor[0] !== undefined) ? this.$editor[0].ownerDocument.body : document.body;
```

Comment:
- `this.$editor[0] ?` is enough no?
- No need in saving the variable in `this.` I think ? Just a normal variable is enough.